[Unity] Fix typo in `get_cuda_target`

cyx-6 opened the pull request:
This PR fixes the typo in `get_cuda_target`, which is used to find the `nvidia-smi` command.

YuchenJin reviewed (approved):
LGTM! Thank you @electriclilies for the work!

cyx-6 committed:
[Unity] Fix typo in `get_cuda_target`

This PR fixes the typo in `get_cuda_target`, which is used to find the `nvidia-smi` command.
